src/test/java/pages/HomePage.java

```diff
@@ -4,42 +4,35 @@
 import org.openqa.selenium.WebElement;
 import org.openqa.selenium.support.FindBy;
 
-public class HomePage extends BasePage{
-
-
-  String url = "https://www.webdriveruniversity.com/";
+public class HomePage extends BasePage {
+  final String URL = "https://www.webdriveruniversity.com/";
 
   @FindBy(id = "dropdown-checkboxes-radiobuttons")
-  private WebElement dropdownLinks;
+  private WebElement dropdownLink;
 
   @FindBy(css = "#popup-alerts")
-  private WebElement popupAndAlertsLink;
-
-  @FindBy(xpath = "//*[@id=\"button1\"]")
-  private WebElement popupButton;
-
+  private WebElement popupAndAlertLink;
 
+  @FindBy(xpath = "(//*[@id='page-object-model'])[last()]")
```

Comment:
Not a good xPath. Try to use specific attributes(text?). Because, if tomorrow some element with id 'page-object-model' will be added, your Xpath will not work.